bullyBot.py

```diff
@@ -0,0 +1,47 @@
+#bullyBot.py
+import os
+import discord
+from discord.ext import commands
+from dotenv import load_dotenv
+
+load_dotenv()
+TOKEN = os.getenv('DISCORD_TOKEN')
+GUILD = os.getenv('DISCORD_GUILD')
+
+bot = commands.Bot(command_prefix='!')
+
+@bot.event
+async def on_ready():
+    for guild in bot.guilds:
+        if guild.name == GUILD:
+            break
+
+    print(
+        f'{bot.user.name} is connected to the following guild:\n'
+        f'{guild.name}(id: {guild.id})'
+    )
+
+@bot.command(name='bully', help='Creates a new channel for bullying a new member')
+async def on_message(ctx, victim):
```

Comment:
What happens when a message comes in with the wrong number of arguments? We should display a help dialog.

Reply:
Help dialog added.

Reply:
+1